### PR TITLE
vcvarsall: add page

### PR DESCRIPTION
**Pull request by aminya**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** Any

### CODE REVIEW

File: pages/windows/vcvarsall.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# vcvarsall
+
+> Setup the environment variables required for using the Microsoft Visual Studio tools from the command line.
+> The path of `vcvarsall.bat` for a certain Visual Studio installation can be found using `vswhere`.
+> More information: <https://docs.microsoft.com/en-us/cpp/build/building-on-the-command-line>.
```
Comment (navarroaxel) [inline pages/windows/vcvarsall.md:5]: We don't use the `i18n` part for Microsoft variables
```suggestion
> More information: <https://docs.microsoft.com/cpp/build/building-on-the-command-line>.
```

**Review by sbrl — APPROVED** [review]
Wow, this looks really useful for Windows users. Thanks!

File: pages/windows/vcvarsall.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# vcvarsall
```
Comment (sbrl) [inline pages/windows/vcvarsall.md:1]: Wait a moment. This has to *exactly* match the name of the command. Same goes for the filename.

**Review by sbrl — CHANGES_REQUESTED** [review]

File: pages/windows/vcvarsall.md (hunk 2 above)
Comment (aminya) [reply pages/windows/vcvarsall.md:1]:
```suggestion
# vcvarsall.bat
```

Comment (aminya) [reply pages/windows/vcvarsall.md:1]: Do you specify exe for all the Windows commands? If not, I should remove bat here too

Comment (navarroaxel) [reply pages/windows/vcvarsall.md:1]: But in Windows `.bat` is optional, I think we shouldn't use `.bat` in the examples.

Comment (aminya) [reply pages/windows/vcvarsall.md:1]: Fixed in https://github.com/tldr-pages/tldr/pull/7439/commits/cd8a83afd56368e307ab948627f6f47f328d1970

**Review by sbrl — APPROVED** [review]
Thanks, @aminya!

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**